### PR TITLE
add ruling

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ To run tests:
 yarn test
 ```
 
+To run ruling:
```
Comment (vilchik-elena) [inline README.md]: @stas-vilchik I would be nice to have some description for "ruling", what it's supposed to test

File: README.md [hunk 2]
```diff
@@ -21,4 +26,4 @@ And finally to run tests with coverage:
 yarn test -- --coverage
 ```
 When you run tests with coverage, the `coverage/` directory will be created at the root. You can
-open the web version of the coverage report `coverage/lcov-report/index.html` to see which lines are covered by tests.
\ No newline at end of file
+open the web version of the coverage report `coverage/lcov-report/index.html` to see which lines are covered by tests.
```
Comment (vilchik-elena) [inline README.md]: @stas-vilchik why this line in diff? I don't see any diff

Comment (stas-vilchik) [inline README.md]: I added a new line in the end

File: README.md (hunk 1 above)
Comment (stas-vilchik) [inline README.md]: do we have one somewhere?

**Comment by stas-vilchik** [comment]
currently line numbers are zero based, would be nice to increase all of them by 1

**Comment by vilchik-elena** [comment]
LGTM

**Pull request merged**